core/commands/log_graph.py

```diff
@@ -8,7 +8,7 @@
 
 COMMIT_NODE_CHAR = "●"
 COMMIT_NODE_CHAR_OPTIONS = "●*"
-COMMIT_LINE = re.compile("^[%s][ /_\|\-.]*([a-z0-9]{3,})" % COMMIT_NODE_CHAR_OPTIONS)
+COMMIT_LINE = re.compile("^[\| ]*[%s][ /_\|\-.]*(?P<commit_hash>[a-z0-9]{3,})" % COMMIT_NODE_CHAR_OPTIONS)
```

Comment:
I think you should use 
GRAPH_CHAR_OPTIONS = " /_\|\-."
```
COMMIT_LINE = re.compile("^[%s]*[%s][%s]*(?P<commit_hash>[a-z0-9]{3,})" % COMMIT_LINE COMMIT_NODE_CHAR_OPTIONS COMMIT_LINE)
```
![with --no-color](https://user-images.githubusercontent.com/3492040/27353996-5ce0b2a2-5606-11e7-8011-0bcb92b8e531.png)
You can get all kind of strange graphs with multimerge.